lib/Sema/SemaBounds.cpp

```diff
@@ -1827,6 +1828,16 @@ namespace {
       // Is R a subrange of this range?
       ProofResult InRange(BaseRange &R, ProofFailure &Cause, EquivExprSets *EquivExprs,
                           std::pair<ComparisonSet, ComparisonSet>& Facts) {
+
+        // In assignments dest <-- src, this relation should hold: range_dest \subseteq range_src
+        // range_dest == {} always passes this test, however we are disallowing empty ranges
+        // for _Array_ptr (upperBound <= lowerBound). With the exception that
+        // upperBound == lowerBound is allowed for _Nt_array_ptr.
+        if ( R.IsEmpty() ) {
```

Comment:
You can remove the spaces inside the if clause.

``
if ( R.IsEmpty() ) --> if (R.IsEmpty())
``